controller/space_areas.go

```diff
@@ -46,7 +47,7 @@ func (c *SpaceAreasController) List(ctx *app.ListSpaceAreasContext) error {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
 		for _, a := range areas {
-			logrus.Info("Found space area with ID=", a.ID.String())
+			log.Info(ctx, map[string]interface{}{"area_id": a.ID.String()}, "Found space area with id %s", a.ID.String())
```

Comment:
if you call `fmt.Sprintf` or alike inside of `log.Info` to format the message, you don't need to call `a.ID.String()` manually. the `fmt` functions do that automatically for you for all objects that implement the `Stringer` interface ;)